From 559cb166601ab9e6fc72c887ef17cd4663608f88 Mon Sep 17 00:00:00 2001
From: Jean Trivelly <jean.trivelly@intel.com>
Date: Wed, 17 Oct 2012 15:27:37 +0200
Subject: [PATCH 49/78] wifi: multi-vendor enabling in mainline

BZ: 60525

Mainline needs to support increasing number of devices.
Each device uses its own hardware mix.
This requires more and more scalability and flexibility.
This is especially true for connectivity where different
manufacturer might be involved.

In order to achieve that, these patches introduces a new way
of managing device configuration allowing us to easily decline
various hardware setup per device.

Change-Id: I3140951428b38a19b23c32cf8076e3ffa064bf92
Signed-off-by: Jean Trivelly <jean.trivelly@intel.com>
Reviewed-on: http://android.intel.com:8080/70701
Reviewed-by: Garcia, JeremieX <jeremiex.garcia@intel.com>
Reviewed-by: Bachot, Jean-michel <jean-michel.bachot@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/78006
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wl12xx-compat/AndroidWl12xxCompat.mk |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/wl12xx-compat/AndroidWl12xxCompat.mk b/wl12xx-compat/AndroidWl12xxCompat.mk
index f980d96..b9fea9b 100644
--- a/wl12xx-compat/AndroidWl12xxCompat.mk
+++ b/wl12xx-compat/AndroidWl12xxCompat.mk
@@ -1,6 +1,7 @@
-# This makefile is included from vendor/intel/*/AndroidBoard.mk.
+# This makefile is included from vendor/intel/common/wifi/WifiRules.mk.
 
 .PHONY: build_wl12xx-compat
 build_wl12xx-compat: build_kernel
 	TARGET_TOOLS_PREFIX="$(ANDROID_BUILD_TOP)/$(TARGET_TOOLS_PREFIX)" vendor/intel/support/wl12xx-compat-build.sh -c $(TARGET_DEVICE)
 
+$(PRODUCT_OUT)/ramdisk.img : build_wl12xx-compat
-- 
1.7.9

